tests/data/dot_gitignore_without_isort

```diff
@@ -0,0 +1,3 @@
+.pre-commit-config.yaml
+.flake8
+.mypy.ini
```

Comment:
It might be simpler to have one dot_gitignore with everything. Unless I am missing something, it is harmless to git-ignore files that don't exist.

Reply:
I agree though that would require changes to the script as well, since now it only adds those file names that were specified by the user.

Reply:
I agree that `isort` should be used everywhere but until this very moment it still is optional so we cannot just enforce it via this package. So I will bring it up at standup next week.

Yes, if `mypy` is added later, it will be added to .gitignore as well. The script does not remove entries so that would argue for adding everything anyway. Still, I prefer to leave this as is for now. Let's discuss this as well at standup next week and I will open a new ticket to modify all behavior in one go.